guide/marketplace/wallet.md

```diff
@@ -0,0 +1,101 @@
+# Wallet
+
+The wallet contains the Ethereum accounts that you will be able to use to interact with the Marketplace.
+
+The wallet holds the private keys of the Ethereum accounts and never expose it.
+
+::: tip Backup everything!
```

Comment:
@antho1404 can I use other color for the this box? I would like a red or orange color

Reply:
`::: warning` or `::: error` should work (not sure about the error, it might be alert)